digdag-core/src/main/java/io/digdag/core/log/LocalFileLogServerFactory.java

```diff
@@ -126,7 +129,24 @@ protected void listFiles(String dateDir, String attemptDir, FileMetadataConsumer
         {
             Path path = getPrefixDir(dateDir, attemptDir).resolve(fileName);
             try (InputStream in = Files.newInputStream(path)) {
-                return ByteStreams.toByteArray(in);
+                if (path.toString().endsWith(LogFiles.LOG_PLAIN_TEXT_FILE_SUFFIX)) {
+                    ByteArrayOutputStream byteOutput = new ByteArrayOutputStream();
```

Comment:
These codes can be written simpler like this.
```
    try (ByteArrayOutputStream byteOutput = new ByteArrayOutputStream();
            GZIPOutputStream gzipOutput = new GZIPOutputStream(byteOutput)) {
        ByteStreams.copy(in, gzipOutput);
        return byteOutput.toByteArray();
    }
```

Reply:
I tried your code but always returned empty response from API server so i added ` gzipOutput.close()` before return byte array.

It's OK?

Reply:
Oops. 👍 

Maybe `gzipOutput.finish()` might be slightly better in terms of understanding the intension from the code.

Reply:
Thanks, changed it.